### PR TITLE
libprotobuf 3.19.6

### PR DESCRIPTION
**Pull request by sh-shahrokhi**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ source:
     folder: third_party/googletest
 
 build:
-  number: 0
+  number: 1
```
Comment (xylar) [inline recipe/meta.yaml:25]:
```suggestion
  number: 0
```
The build number should only be increased for a new build of the same version.

**Comment by sh-shahrokhi** [comment]
@xylar pleases check this one.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -112,4 +112,3 @@ extra:
     - wesm
     - hajapy
     - xhochy
-    - h-vetinari
```
Comment (xylar) [inline recipe/meta.yaml:115]: Since this isn't going to `main` it doesn't matter but let's not remove any maintainers.  It sends a bad vibe.

Comment (sh-shahrokhi) [reply recipe/meta.yaml]: Thanks

File: recipe/meta.yaml (hunk 1 above)
Comment (sh-shahrokhi) [reply recipe/meta.yaml]: Thank you

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@sh-shahrokhi, you need to do the actual rerender step (so please don't check that box until you actually post the command).

**Comment by xylar** [comment]
Okay, I think we're good now.  I'll merge once tests pass.

**Comment by sh-shahrokhi** [comment]
> Okay, I think we're good now. I'll merge once tests pass.

It passed. Thank you

**Pull request merged**